pkg/discovery/network/openshift4.go

```diff
@@ -43,8 +44,10 @@ func discoverOpenShift4Network(dynClient dynamic.Interface) (*ClusterNetwork, er
 
 	cr, err := crClient.Get("default", metav1.GetOptions{})
 	if err != nil {
-		klog.Info("Attempted network discovery for OpenShift4, no clusternetworks CRD")
-		return nil, nil
+		if apierrors.IsNotFound(err) {
```

Comment:
I think that missing CRD could be another type of error we may need to check.

Reply:
Good point - I'll check on that.

Reply:
I verified that it still returns `IsNotFound` error if the resource type doesn't exit. That's also evident b/c all the e2e tests succeeded :smile: